man/systemd.network.xml

```diff
@@ -3060,6 +3060,15 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RetransmitNeighborSolicitationSec=</varname></term>
+
+        <listitem><para>Takes a timespan. Configures the minimum time that the client should use between retransmitted
+        Neighbor Solicitation message that is sent in the Router Advertisement message. Takes a timespan
```

Comment:
Configures the retransmit time, used by clients to retransmit Neighbor Solicitation messages on address resolution and the Neighbor Unreachability Detection algorithm.